features/other_a/autocomplete.feature

```diff
@@ -98,15 +98,19 @@ Feature: Display autocomplete for tags
       And the pseud autocomplete should not contain "new_user"
       And the pseud autocomplete should contain "different_user"
       And the pseud autocomplete should contain "funny (different_user)"
-    When I change my username to "funny"
-    Then the pseud autocomplete should not contain "funny (different_user)"
-      And the pseud autocomplete should contain "funny"
-      And the pseud autocomplete should contain "different_user (funny)"
-    When I try to delete my account as funny
+    When I try to delete my account as different_user
     Then a user account should not exist for "funny"
       And the pseud autocomplete should not contain "funny"
       And the pseud autocomplete should not contain "different_user (funny)"
 
+  Scenario: Pseuds should be listed in autocomplete when username changes to pseud
```

Comment:
I think we can remove this scenario. We're testing much more for autocomplete here. Plus there's already:

https://github.com/otwcode/otwarchive/blob/d42d1bbe5b6ff49fa4bf75e6a5b11738341eb52f/features/users/user_rename.feature#L95